package.json

```
@@ -53,7 +53,7 @@
     "@onflow/types": "^0.0.5",
     "elliptic": "^6.5.4",
     "esm": "^3.2.25",
-    "flow-cadut": "^0.1.10",
+    "flow-cadut": "0.1.11-alpha.13",
```

`0.1.11` version is released now, so we can update this package
```suggestion
    "flow-cadut": "0.1.11",
```

---

updated!